readme.md

```diff
@@ -117,6 +119,20 @@ Note: executing the command `javaflow Person.java` without the `types.yml` file
 to the program failing due to the `Ssn.java` file not being found. `javaflow` does not look
 for additional files which where not supplied as input.
 
+### Verifiers ###
+
+In case you want to verify that you have specified mappings for all types,
+that the DTOs have getters for all fields,
+or that the type of the fields are same as the return types of their getters,
+`javaflow` offers the possibility to add verifiers.
+
+Available verifiers:
+- `MemberFieldsPresentVerifier`, active by default, verifies that there are no missing types referenced in the list of types.
+- `ClassGetterNamingVerifier`, can be activated by passing the `--verifyGetters` flag in the CLI, verifies if the given types all have:
+    - the same number of fields and getters,
+    - the same type in the field definitions as a return type in the corresponding getters and
```

Comment:
I propose: _the same type in the field definitions as **the** return type in the corresponding **getter** and_